profdev/services.html

```diff
@@ -9,41 +9,39 @@
 
 <a id="Top"></a>
 <div class="cover" style="background-image: url({{page.cover-pic}})">
-  {% include section-header.html title=page.title tagline="Professional Development Services" color="#fff" class="big" %}
-  <a class="button bottom-top-top" href="/profdev/oh-signup">Sign Up Now</a>
+	{% include section-header.html title="Advising" tagline="Professional Development Services" color="#fff" class="big" %}
```

Comment:
This is true. I made this change because I thought "Professional Services appointment" sounded weird. Also the tagline became redundant. I am reverting this now.